bin/run.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/sh
 mkdir -p $BACKUP_DATA_DIR
 
+echo "Checking database connection to host $POSTGRESQL_SERVICE_HOST..."
+export PGPASSWORD=$POSTGRESQL_ADMIN_PASSWORD
+psql --username=$POSTGRESQL_ADMIN_USER --host=$POSTGRESQL_SERVICE_HOST --port=$POSTGRESQL_SERVICE_PORT -d postgres -t -c "SELECT version();"
+if [ $? -eq 0 ]; then
+  echo "Connection OK."
+else
+  echo "Connection FAILED. Continuing anyway..."
```

Comment:
souldn't we fail in that case ?
if the db credentials or host information are wrong, it would be ok if the container could not start. This check would make much more sense if it fails with invalid config or down db container ?